src/modules/30-secrets/components/CreateUpdateSecret/views/VaultFormFields.tsx

```diff
@@ -26,9 +27,31 @@ const VaultFormFields: React.FC<VaultFormFieldsProps & FormikContextProps<any>>
   formik,
   type,
   editing,
-  readonly
+  readonly,
+  secretManagerType
 }) => {
   const { getString } = useStrings()
+  const [regions, setRegions] = React.useState<SelectOption[]>([])
+  const { showError } = useToaster()
+
+  const { data: regionData, error, refetch } = useGetGcpRegions({ lazy: true })
+  if (error) {
+    showError(error.message)
+  }
+  React.useEffect(() => {
+    if (regionData?.data && regionData?.data.length) {
+      const regionValues = (regionData?.data || []).map(region => ({
+        value: region,
+        label: region
+      }))
+      setRegions(regionValues as SelectOption[])
+    }
+  }, [regionData])
+  React.useEffect(() => {
+    if (secretManagerType === 'GcpSecretManager') {
+      refetch()
+    }
+  }, [])
```

Comment:
This is crying out for some whitespace! Having no line breaks makes this very difficult to visually parse at a glance.
```suggestion
  const { data: regionData, error, refetch } = useGetGcpRegions({ lazy: true })
  
  if (error) {
    showError(error.message)
  }
  
  React.useEffect(() => {
    if (regionData?.data && regionData?.data.length) {
      const regionValues = (regionData?.data || []).map(region => ({
        value: region,
        label: region
      }))
      setRegions(regionValues as SelectOption[])
    }
  }, [regionData])
  
  React.useEffect(() => {
    if (secretManagerType === 'GcpSecretManager') {
      refetch()
    }
  }, [])
  
```